library/bdImage/bdApi/ControllerApi/Post.php

```diff
@@ -26,7 +27,7 @@ public function actionGetEditorConfig()
                         $data[bdImage_Helper_Data::IMAGE_URL] === $threadImageData[bdImage_Helper_Data::IMAGE_URL] ? true : false,
                     'image_is_cover' =>
                         ($data[bdImage_Helper_Data::IMAGE_URL] === $threadImageData[bdImage_Helper_Data::IMAGE_URL]
-                            & $threadImageData['is_cover']) ? true : false
+                            & (!empty($threadImageData['is_cover']) ? $threadImageData['is_cover'] : false)) ? true : false
```

Comment:
Nếu đã `!empty` rồi thì cần gì check lại `$threadImageData['is_cover']`.

Còn nếu muốn check 2 lần thì lần 1 phải xài `isset` chứ?

Reply:
e có fix lại rồi theo a xem thử nha

Reply:
Ủa sao giờ vừa `isset` vừa `empty`? :(

Reply:
lại thử a ơi :">

Reply:
Nếu đang xài `xxx ? true : false` thì bỏ luôn đoạn `? true : false` được nha chế. Vì nó vô nghĩa đó...

Với cả toán thử boolean AND thì phải là `&&` nha. Nếu xài `&` nó là bitwise AND.

Reply:
noted tks a Sơn